src/io/rkn/conformity.clj

```diff
@@ -56,32 +56,53 @@
   (or (some #(and (has-attribute? db %) %) [:conformity/conformed-norms default-conformity-attribute])
       :conformity/conformed-norms))
 
+(defn last-tx-instant
+  "Returns a value of the :db/txInstant attribute of the last transaction."
+  [db]
+  (let [tx (-> db (d/basis-t) (d/t->tx))]
+    (ffirst (q '[:find ?inst
+                 :in $ ?tx
+                 :where [?tx :db/txInstant ?inst]]
+              db tx))))
+
+(defn with-tx-instant
+  "If instant is not nil, add it as the :db/txInstant attribute of transaction."
+  [instant tx-data]
+  (cond->> tx-data
```

Comment:
Would prefer using if because the only branch is whether instant is provided/not